core/src/game_state.rs

```diff
@@ -25,7 +25,7 @@ pub enum MessageVariant {
     NumFriendsSet { num_friends: Option<usize> },
     GameModeSet { game_mode: GameModeSettings },
     TookBackPlay,
-    PlayedCards { cards: Vec<Card> },
+    PlayedCards { player_name: String, cards: Vec<Card> },
```

Comment:
hm. You should have this information from `actor`, right? I don't think it needs to be duplicated here.

Reply:
the list of players isn't present in the message. So we'd either need to combine the Message with the GameState, assuming that it is in a state with players, or add it here